### ISSUE TITLE
Error detected while processing LspAttach Autocommands for "*":

### ISSUE THREAD

**Issue by emrhnpla02** [nvimdev/lspsaga.nvim#1199]
### Describe the bug

 ```
Error detected while processing LspAttach Autocommands for "*":
Error executing lua callback: ...share/nvim/lazy/lspsaga.nvim/lua/lspsaga/symbol/head.lua:25: unexpected event
stack traceback:
	[C]: in function 'nvim_create_autocmd'
	...share/nvim/lazy/lspsaga.nvim/lua/lspsaga/symbol/head.lua:25: in function 'buf_watcher'
	...share/nvim/lazy/lspsaga.nvim/lua/lspsaga/symbol/head.lua:180: in function <...share/nvim/lazy/lspsaga.nvim/lua/lspsaga/symbol/head.lua:165>
	[C]: in function 'nvim_exec_autocmds'
	...stalls/neovim/nightly/share/nvim/runtime/lua/vim/lsp.lua:1687: in function '_on_attach'
	...stalls/neovim/nightly/share/nvim/runtime/lua/vim/lsp.lua:1454: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>
```
I get error above when I open any file that starts a lsp client

here's my config

```lua
local status_ok, lspsaga = pcall(require, "lspsaga")
if not status_ok then
  vim.notify("Lspsaga not found!")
  return
end

lspsaga.setup({
  lightbulb = {
    enable = true,
    enable_in_insert = true,
    sign = true,
    sign_priority = 40,
    virtual_text = false,
  },
  finder = {
    edit = "l",
    vsplit = "v",
    split = "s",
    quit = { "q", "<ESC>" },
  },
  code_action = {
    num_shortcut = true,
    keys = {
      quit = "<ESC>",
      exec = "<CR>",
    },
  },
  rename = {
    quit = "q",
  },
  symbol_in_winbar = {
    enable = false,
  },
  outline = {
    keys = {
      jump = "h",
      expand_collapse = "l",
      quit = "<ESC>",
    },
  },
  beacon = {
    enable = true,
    frequency = 10,
  },
  ui = {
    border = "rounded",
    kind = require("catppuccin.groups.integrations.lsp_saga").custom_kind(),
  },
})

keymap("n", "<C-o>", "<cmd>Lspsaga outline<CR>", opts)
keymap("n", "gd", "<cmd>Lspsaga goto_definition<CR>", opts)
keymap("n", "gD", "<cmd>Lspsaga peek_definition<CR>", opts)
keymap("n", "gl", "<cmd>Lspsaga show_line_diagnostics<CR>", opts)
```

### Steps to reproduce

1- Open any file that has a configured lsp client

### Expected behavior

To not throw an error i guess

### Neovim version (nvim -v)

NVIM v0.10.0-dev-664+g0ce391086

### lspsaga commit

f4bfc44

### Terminal name/version

kitty 0.28.1

**Comment by UrNightmaree** [comment]
After looking at the source code, Neovim is complaining about invalid autocmd event, which is `LspNotify`. `LspNotify` seems to be available in nightly version of Neovim, but it's weird since it shouldn't complain about invalid autocmd even if you're using nightly version.

**Comment by UrNightmaree** [comment]
I found the solution, you'll need to build neovim/neovim@63b3408551561127f7845470eb51404bcd6f547b in order to fix this problem.

**Comment by glepnir** [comment]
build with recently commit

**Comment by rodhash** [comment]
It seems this "issue" started on commit `2f9d768`, building latest neovim fixed too.

**Comment by UrNightmaree** [comment]
You don't need to build latest Neovim just to fix this error, #1201 should fix this issue.
